pyproject.toml

```diff
@@ -22,6 +22,12 @@ requires-python = ">=3.7"
 dependencies = [
     "flask>=2.2",
     "cachelib",
+    "redis",
+    "pymongo",
+    "python3-memcached",
+    "Flask-SQLAlchemy",
+    # Testing
+    "nose",
```

Comment:
Also, this is a dev dependency, not a runtime dependency. Someone who just wants to use this library won't need it.

Reply:
Addressed in the latest commit